src/streamlink_cli/argparser.py

```diff
@@ -627,8 +598,13 @@ def build_parser():
 
         You will be prompted if the file already exists.
 
-        The formatting variables available for the --title option may be used.
+        Please see the "Metadata variables" section of Streamlink's CLI documentation for all available metadata variables.
+
         Unsupported characters in substituted variables will be replaced with an underscore.
+
+        Example:
+
+            %(prog)s --record-and-pipe "~/recordings/{author}/{category}/{id}-{time:%Y%m%d%H%M%S}.ts" <URL> [STREAM]
```

Comment:
I don't know if it's a Netlify thing only, and it's certainly very minor, but I see that different CSS is applied to `and` in `--record-and-pipe` here.  I thought it was eyes at first!  Technically, that's nothing to do with the content of this PR though, but it's just something I noticed and thought I would mention.

Reply:
It's because of the custom argparse extension which tries to build ReST from certain text patterns used in the argparse help texts:
https://github.com/streamlink/streamlink/blob/3309ed0620fa4aedeee218a48996d8284a90bf4b/docs/ext_argparse.py#L22
https://github.com/streamlink/streamlink/blob/3309ed0620fa4aedeee218a48996d8284a90bf4b/docs/ext_argparse.py#L72-L73

Text indented by at least 2 spaces gets turned into a literal-block which then gets highlighted by Sphinx automatically when there's no block type specified:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks

There are a couple of other examples in the docs where it's parsing it incorrectly, but we can't use full ReST syntax in the argparse help texts because of the `--help` output and the man page, so it's a bit difficult to fix this.

Reply:
It's so trivial it was not worth worrying about anyway, but more so if the fix is tricky.  I just wasn't sure if it had been noticed before.